siteimprove/admin/js/siteimprove.js

```diff
@@ -119,43 +119,40 @@
                   }
               });
           }
-    
-          var wrapTag = $("<span class='si-highlight'></span>");
-    
-          // Store the original content for all elements that are currently highlighted
-          // $this.data() is jQuery internal cache storage
-          $(".si-highlight").each(function() {
-              $(this).data('original-content', $(this).html());
-          });
-    
-          // Restore the original content where the previous highlight span was applied
-          $(".si-highlight").each(function() {
-              $(this).replaceWith($(this).data('original-content'));
-          });
 
-          // We handle body tag highlight specificially, so we also need to remove it specificially from HTML element
-          if($("body").hasClass("si-full-highlight")) {
-            $("body").removeClass("si-full-highlight");
+          function resetHighlights() {
+            $(".si-highlight").each(function() {
```

Comment:
Maybe add a comment here about what this does with the child replacement of the span